helm/defectdojo/templates/gke-managed-certificate.yaml

```diff
@@ -1,9 +1,22 @@
-{{- if .Values.gke.useManagedCertificate }}
+{{- if .Values.gke.useManagedCertificate | and (.Capabilities.APIVersions.Has "networking.gke.io/v1") }}
```

Comment:
Thanks for this. It is a bit hard part for testing (if you do not have access to GKE)